web/pgadmin/authenticate/oauth2.py

```diff
@@ -151,6 +151,26 @@ def login(self, form):
                 current_app.logger.exception(error_msg)
                 return False, gettext(error_msg)
 
+        additinal_claims = None
+        if 'OAUTH2_ADDITIONAL_CLAIMS' in self.oauth2_config[
+                self.oauth2_current_client]:
+            additinal_claims = self.oauth2_config[
+                self.oauth2_current_client
+            ]['OAUTH2_ADDITIONAL_CLAIMS']
+
+        (valid, reason) = self.__is_additional_claims_valid(profile,
+                                                            additinal_claims)
+
+        if not valid:
+            return_msg = "Your user it's not authorized to access" \
```

Comment:
return_msg = gettext("The user is not authorized to login "
                                 "based on the claims in the profile. "
                                 "Please contact your administrator.")